sdk/python/flet/container.py

```diff
@@ -359,29 +360,36 @@ def content(self, value: Optional[Control]):
 
     # shape
     @property
-    def shape(self):
-        return self._get_attr("shape")
+    def shape(self) -> Union[BoxShape, BoxShapeString]:
+        return self.__shape
 
     @shape.setter
+    def shape(self, value: Union[BoxShape, BoxShapeString]):
+        self.__shape = value
+        if isinstance(value, BoxShape):
+            self._set_attr("shape", value.value)
+        else:
+            self.__set_shape(value)
+
     @beartype
-    def shape(self, value: Optional[BoxShape]):
-        self._set_attr("shape", value.value if value is not None else None)
+    def __set_shape(self, value: BoxShapeString):
+        self._set_attr("shape", value)
```

Comment:
I am just from noticing the importance of this part, and similar parts of this PR.
![image](https://user-images.githubusercontent.com/98978078/213703993-5ddbe02a-0e37-4e4a-a9c6-781e254988d0.png)
This should _not_ be a normal behavior @FeodorFitsner. One should be able to pass either a string or a `BoxShape `. At the moment, only a `BoxShape` is accepted, while strings give an error as shown above.
What do you think?

Reply:
Hey @FeodorFitsner